junit-platform-engine/src/main/java/org/junit/platform/engine/support/hierarchical/NodeTestTask.java

```diff
@@ -165,13 +165,25 @@ private void cleanUp() {
 
 	private void reportCompletion() {
 		if (throwableCollector.isEmpty() && skipResult.isSkipped()) {
+			try {
+				node.nodeSkipped(context, testDescriptor, skipResult);
+			}
+			catch(Throwable t) {
+				//swallow
+			}
 			listener.executionSkipped(testDescriptor, skipResult.getReason().orElse("<unknown>"));
 			return;
 		}
 		if (!started) {
 			// Call executionStarted first to comply with the contract of EngineExecutionListener.
 			listener.executionStarted(testDescriptor);
 		}
+		try {
+			node.nodeFinished(context, testDescriptor, throwableCollector.toTestExecutionResult());
+		}
+		catch(Throwable t) {
+			//swallow
```

Comment:
Swallowing `Throwable` is not allowed in JUnit 5. We need to rethrow it if it's _blacklisted_.